language/diem-tools/writeset-transaction-generator/src/main.rs

```diff
@@ -108,28 +123,33 @@ fn main() -> Result<()> {
             first_release,
             diem_version,
         } => {
-            let release_modules = diem_framework_modules();
+            let release_name = opt
+                .output
+                .clone()
+                .expect("Empty output path provided")
+                .file_stem()
+                .expect("Path should be a file")
+                .to_str()
+                .expect("Path name should be able to convert to string")
+                .to_owned();
+
+            let release_modules = diem_framework_modules(release_name.as_str());
```

Comment:
I guess this comes back to the question above as well, but just want to make sure we're releasing the Move stdlib modules at the same time as the new diem framework modules

Reply:
I guess this is a question for @vgao1996 ? To me the Move stdlib are changed a lot less frequently so it's not as a big issue as the diem framework modules.